asm/cluster/cluster.yaml

```diff
@@ -47,5 +47,7 @@ spec:
     useIpAliases: true
   masterAuth:
     username: admin
-  network: default
-  subnetwork: default
+  network: default  # {"type":"string","x-kustomize":{"setter":{"name":"gcloud.compute.network"}}}
+  subnetwork: default # {"type":"string","x-kustomize":{"setter":{"name":"gcloud.compute.subnetwork"}}}
+  releaseChannel:
+    channel: REGULAR
```

Comment:
Release Channel is already added before. See L30-L31.

Reply:
Fixed